validphys2/src/validphys/pseudodata.py

```diff
@@ -129,6 +130,7 @@ def make_replica(
     replica_mcseed,
     dataset_inputs_sampling_covmat,
     sep_mult,
+    fitbasis="NN3IC",
```

Comment:
?

Reply:
I guess, here, you are mainly referring to https://github.com/NNPDF/nnpdf/issues/2041?

Reply:
I don't understand why this change is necessary, it is not used in the function is it?

Reply:
Ah, sorry! That is indeed no longer needed. It was there before to allow negative pseudodata for POL but now it is encoded in the `process_type`. I will remove it in the next batch while taking care of the other comments.